inst/www/shinyTable.js

```diff
@@ -320,10 +320,9 @@ Shiny.addCustomMessageHandler('htable-change', function(data) {
   
   var tbl = $el.handsontable('getInstance');
   for( var i = 0; i < data.changes.length; i++){
-    var change = data.changes[i];
     tbl.setDataAtCell(
-      parseInt(change.row), 
-      parseInt(change.col),
+      parseInt(data.changes.row[i]), 
+      parseInt(data.changes.col[i]),
       change.new,
```

Comment:
Well, maybe I don't see the difference between 
```javascript
    for( var i = 0; i < data.changes.length; i++){
      var change = data.changes[i];
      tbl.setDataAtCell(
        parseInt(change.row), 
        parseInt(change.col),
...
```
and
```javascript
    for( var i = 0; i < data.changes.length; i++){
      tbl.setDataAtCell(
        parseInt(data.changes.row[i]), 
        parseInt(data.changes.col[i]),
...
```

But I just took the code from @jrowen's pull request ang put it into this new PR. And approach with standalone ``var change`` declaration is more readable. I'll put it back.